### PR TITLE
Add Swapfiets for multiple countries

### LINKED ISSUE #1174: Add the different companies running *Swapfiets*
Swapfiets is run by different companies, depending on the country. In #1165 I've added the german one. 
There are still others that can be, added, such as: 
- [ ] Swapfiets B.V (https://swapfiets.com/nl-NL/privacy)
- [ ] Swapfiets Demark ApS (https://swapfiets.com/da-DK/privacy)
- [ ] Swapfiets België BVBA (https://swapfiets.com/nl-BE/privacy)
- [ ] Swapfiets Spain S.l (https://swapfiets.com/en-ES/privacy) 
- [ ] Swapfiets Italia S.r.l (https://swapfiets.com/it-IT/privacy)
- [ ] Swapfiets Limited and Swapservices B.V. (https://swapfiets.com/en-GB/privacy)
- [ ] Swapfiets Austria GmbH (https://swapfiets.com/de-AT/privacy)

### PR DESCRIPTION
**Pull request by moriarty-jim**
Fixes #1174 

Countries: at, uk, it, es, be, dk, nl

### CODE REVIEW

File: companies/swapfiets-be.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "swapfiets-be",
+    "relevant-countries": [
+        "be"
+    ],
+    "name": "Swapfiets België BVBV",
+    "address": "Mechelseplein 10\n2000 Antwerpen\nBelgium",
```
Comment (mal-tee) [inline companies/swapfiets-be.json:7]: Can you add a source for that address? Or did I miss sth.?

File: companies/swapfiets-dk.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "swapfiets-dk",
+    "relevant-countries": [
+        "dk"
+    ],
+    "name": "Swapfiets Demark ApS",
+    "address": "Nordre Fasanvej 17\n2000 Frederiksberg\nDenmark",
```
Comment (mal-tee) [inline companies/swapfiets-dk.json:7]: same here

File: companies/swapfiets-it.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "relevant-countries": [
+        "it"
+    ],
+    "name": "Swapfiets Italia s.r.l.",
+    "address": "Via Adda 16\n20090 Opera MI\nItaly",
+    "phone": "+39 02 8295 8065",
+    "email": "privacy@swapfiets.com",
+    "web": "https://swapfiets.it/",
+    "sources": [
+        "https://swapfiets.it/en-IT/privacy",
```
Comment (mal-tee) [inline companies/swapfiets-it.json:12]: This source lists "Swapfiets B.V" as the responsible entity.

Comment (mal-tee) [reply companies/swapfiets-it.json:12]: But as the phone number and address are in Italy-specific, we can't really merge this. @baltpeter What do you think?

**Review by mal-tee — CHANGES_REQUESTED** [review]
Thank you! I've left some notes.

File: companies/swapfiets-be.json (hunk 1 above)
Comment (moriarty-jim) [reply companies/swapfiets-be.json:7]: done

File: companies/swapfiets-it.json (hunk 3 above)
Comment (baltpeter) [reply companies/swapfiets-it.json:12]: Haven't check the source but I agree. In that case, they should stay separate records. We can have multiple records with the same `name`.

Comment (moriarty-jim) [reply companies/swapfiets-it.json:12]: I think this is a translation/copy-paste error. The [Italian version](https://swapfiets.it/it-IT/privacy) lists Swapfiets Italia

Comment (baltpeter) [reply companies/swapfiets-it.json:12]: That seems likely, good catch! I would prefer to use the Italian source then.

Comment (moriarty-jim) [reply companies/swapfiets-it.json]: 👍 changed it

**Review by mal-tee — APPROVED** [review]
Thank you!

**Pull request merged**